Type 'Observable<boolean | Account>' is not assignable to type 'Observable<boolean | Account>'.

(23,9): error TS90010: Type 'Observable<boolean | Account>' is not assignable to type 'Observable<boolean | Account>'. Two different types with this name exis
t, but they are unrelated.
        Property 'source' is protected but type 'Observable<T>' is not a class derived from 'Observable<T>'

Is it possible run this app from the command line? When I try `dotnet run` on my Mac, I get the following error:

```
fail: Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware[0]
      An unhandled exception has occurred: Call to Node module failed with error: Prerendering failed because of error: Error: Cannot find module '/Users/mraible/dev/kuda/ClientApp/dist/main-server'
          at Function.Module._resolveFilename (module.js:469:15)
          at Function.Module._load (module.js:417:25)
          at Module.require (module.js:497:17)
          at require (internal/module.js:20:19)
          at findBootModule (/private/var/folders/_g/h5m4s5_50175ntlmwkf0hw9h0000gn/T/tmp432QTX.tmp:111:17)
          at findRenderToStringFunc (/private/var/folders/_g/h5m4s5_50175ntlmwkf0hw9h0000gn/T/tmp432QTX.tmp:116:28)
          at renderToStringImpl (/private/var/folders/_g/h5m4s5_50175ntlmwkf0hw9h0000gn/T/tmp432QTX.tmp:75:51)
          at /private/var/folders/_g/h5m4s5_50175ntlmwkf0hw9h0000gn/T/tmpMktFZF.tmp:113:19
          at IncomingMessage.<anonymous> (/private/var/folders/_g/h5m4s5_50175ntlmwkf0hw9h0000gn/T/tmpMktFZF.tmp:132:38)
          at emitNone (events.js:86:13)
      Current directory is: /Users/mraible/dev/kuda
```

--- a/README.md
+++ b/README.md
@@ -59,6 +59,24 @@ export class AppComponent extends AuthPortComponent {
 }
 ```
 
+If you're using a version of Angular < 2.3, extending AuthPortComponent won't work for you. As an alternative, you can inject the `Stormpath` service into your component, subscribe to `stormpath.user$` and implement logout yourself.
+
+```typescript
+import { Account, Stormpath } from 'angular-stormpath';
+...
+export class AppComponent {
+  user$: Observable<Account | boolean>;
+
+  constructor(private stormpath: Stormpath) {
+    this.user$ = this.stormpath.user$;
+  }
+
+  logout(): void {
+    this.stormpath.logout();
+  }
+}
+```
+
 You may also find it useful to view the [demo source](https://github.com/stormpath/stormpath-sdk-angular/blob/master/demo/app.component.ts).
 
 ### Configuration

--- a/README.md
+++ b/README.md
@@ -59,7 +59,7 @@ export class AppComponent extends AuthPortComponent {
 }
 ```
 
-If you're using a version of Angular < 2.3, extending AuthPortComponent won't work for you. As an alternative, you can inject the `Stormpath` service into your component, subscribe to `stormpath.user$` and implement logout yourself.
+If you're using a version of Angular < 2.3, extending AuthPortComponent won't work for you. As an alternative, you can inject the `Stormpath` service into your component, subscribe to `stormpath.user$` and implement `logout()` yourself.
 
 ```typescript
 import { Account, Stormpath } from 'angular-stormpath';

--- a/package.json
+++ b/package.json
@@ -106,10 +106,10 @@
     "webpack-dev-server": "2.2.0",
     "zone.js": "^0.7.4"
   },
-  "dependencies": {
-    "@angular/core": "^2.4.0",
-    "@angular/forms": "^2.4.0",
-    "@angular/http": "^2.4.0",
+  "peerDependencies": {
+    "@angular/core": "^2.0.0",
+    "@angular/forms": "^2.0.0",
+    "@angular/http": "^2.0.0",
     "rxjs": "^5.0.2",
     "ng2-webstorage": "1.4.3",
     "angular2-cookie": "^1.2.6"

